src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+   public String getRandomBall() {
```

Comment:
```suggestion
   public Ball getRandomBall() {
```

lets make this method to return Ball not String